bigtable/client/client_options.h

```diff
@@ -53,6 +53,126 @@ class ClientOptions {
     return *this;
   }
 
+  /**
+   * Set compression algorithm for channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetCompressionAlgorithm()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetCompressionAlgorithm(grpc_compression_algorithm algorithm) {
+    channel_arguments_.SetCompressionAlgorithm(algorithm);
+  }
+
+  /**
+   * Set the grpclb fallback timeout (in ms) for the channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetGrpclbFallbackTimeout()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetGrpclbFallbackTimeout(int fallback_timeout) {
+    channel_arguments_.SetGrpclbFallbackTimeout(fallback_timeout);
+  }
+
+  /**
+   * Set Socket Mutator for channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetSocketMutator()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetSocketMutator(grpc_socket_mutator* mutator) {
+    channel_arguments_.SetSocketMutator(mutator);
+  }
+
+  /**
+   * Set the string to prepend to the user agent.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetUserAgentPrefix()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetUserAgentPrefix(const grpc::string& user_agent_prefix) {
+    channel_arguments_.SetUserAgentPrefix(user_agent_prefix);
+  }
+
+  /**
+   * Set the buffer pool to be attached to the constructed channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetResourceQuota()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetResourceQuota(const grpc::ResourceQuota& resource_quota) {
+    channel_arguments_.SetResourceQuota(resource_quota);
+  }
+
+  /**
+   * Set the max receive message sizes in bytes. -1 means unlimited.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetMaxReceiveMessageSize()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  void SetMaxReceiveMessageSize(int size) {
+    channel_arguments_.SetMaxReceiveMessageSize(size);
+  }
+
+  /**
+   * Set the max send message sizes in milliseconds. INT_MAX means unlimited.
```

Comment:
message size in milliseconds?

Reply:
Good catch, @coryan! I completely glossed over that.

Reply:
It was awkward for me too but this is what is mentioned in code documentation.

Reply:
https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html#ac1fa513191e8104ec57dfd6598297ce5 doesn't say anything about milliseconds; in fact, that method has no comments (it should!).

Reply:
I am going to suggest changing it to bytes, since it can only be that (and parallels receive message size) and go with that. If it's wrong, we can fix it later, but I don't see how it can be anything else.

Reply:
My bad. Checked again. Misread the comment for some other function. Changed to bytes.